src/libraries/System.Text.Json/gen/Reflection/FieldInfoWrapper.cs

```diff
@@ -100,5 +100,7 @@ public override void SetValue(object obj, object value, BindingFlags invokeAttr,
         {
             throw new NotImplementedException();
         }
+
+        public Location? Location => _field.Locations.Length > 0 ? _field.Locations[0] : null;
```

Comment:
Could this be 
```suggestion
        public Location Location => _field.Locations.Length > 0 ? _field.Locations[0] : Location.None;
```
?

Reply:
`null` is more convenient here since we can succinctly do `null`-checks later to determine whether to use a different diagnostic location.